back/src/user/user.js

```diff
@@ -8,14 +8,18 @@ class User {
         this.password = password;
     }
 
+    /**
+     * Add user data to the database
+     */
     addToDatabase() {
-        const conf = require('../conf/conifg.js')
+        const conf = require('../config/config.js')
```

Comment:
We should probably put this at the top of the file instead of inside the method if we're going to use this file for all queries.